client/my-sites/google-my-business/controller.js

```diff
@@ -8,10 +8,21 @@ import React from 'react';
 /**
  * Internal Dependencies
  */
+import GoogleMyBusinessNewAccount from './new-account';
 import GoogleMyBusinessSelectBusinessType from './select-business-type';
 
+export function newAccount( context, next ) {
+	const { params } = context;
+
+	context.primary = <GoogleMyBusinessNewAccount siteId={ params.site_id } />;
```

Comment:
I'd rather get the site id from the store here.
```
const state = context.store.getState();
const siteId = getSelectedSiteId( state );
```
And use the more standard `:site` path section instead of `:site_id` in our route definitions

Reply:
I'm a bit lost when it comes to best practices about routing. There are so many variants in our code, I wished we had a document explaining the one we should use. Let's follow your suggestions 👍 .

Reply:
>  I wished we had a document explaining the one we should use

Well done for volunteering :)